lib/galaxy/files/sources/webdav.py

```diff
@@ -23,11 +23,17 @@ class WebDavFilesSource(PyFilesystem2FilesSource):
 
     def _open_fs(self, user_context=None, opts: Optional[FilesSourceOptions] = None):
         props = self._serialization_props(user_context)
-        use_temp_files = props.pop("use_temp_files", None)
-        if use_temp_files is None:
-            # Default to True to avoid memory issues with large files.
-            props["use_temp_files"] = True
-            props["temp_path"] = props.get("temp_path", tempfile.TemporaryDirectory(prefix="webdav_"))
+
+        # Check if 'use_temp_files' and 'temp_path' are set in the config, otherwise fallback
+        props["use_temp_files"] = getattr(config, 'webdav_use_temp_files', True)
```

Comment:
This seems like a leftover.

Reply:
I think the `use_temp_files` is needed by the `WebDAVFS` when using the `temp_path`.

Reply:
Sorry, i just realized its looking for the config variable. ill push a commit fixing this.